store/CSDK/src/CarbonWriter.cpp

```diff
@@ -110,6 +110,20 @@ void CarbonWriter::withCsvInput(char *jsonSchema) {
     }
 };
 
+void CarbonWriter::withCsvInput() {
```

Comment:
Since this is interface exposed to user, please add comment to describe the function

Reply:
Already describe it in interface,also added in here